dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -795,6 +826,331 @@ def __call__(self, *args, **kwargs):
         return super().__call__(*args, **kwargs)
 
 
+class DPNPBinaryTwoOutputsFunc(BinaryElementwiseFunc):
+    """
+    Class that implements unary element-wise functions with two output arrays.
+
+    Parameters
+    ----------
+    name : {str}
+        Name of the unary function
```

Comment:
```suggestion
        Name of the binary function
```

also in DPNPBinaryFunc